bitnami/thanos/templates/receive/statefulset.yaml

```diff
@@ -71,13 +71,17 @@ spec:
             - --grpc-address=0.0.0.0:10901
             - --http-address=0.0.0.0:10902
             - --remote-write.address=0.0.0.0:19291
-            - --receive.replication-factor=1
+            - --receive.replication-factor={{ .Values.receive.replicationFactor }}
             - --objstore.config=$(OBJSTORE_CONFIG)
             - --tsdb.path=/var/thanos/receive
             - --label={{ .Values.receive.replicaLabel }}="$(NAME)"
             - --label=receive="true"
             - --tsdb.retention=15d
+            {{- if and (eq .Values.receive.replicaCount 1) .Values.receive.config }}
```

Comment:
@Cellebyte There's an error when running this part:
```
Error: template: thanos/templates/receive/statefulset.yaml:80:24: executing "thanos/templates/receive/statefulset.yaml" at <eq .Values.receive.replicaCount 1>: error calling eq: incompatible types for comparison
helm.go:81: [debug] template: thanos/templates/receive/statefulset.yaml:80:24: executing "thanos/templates/receive/statefulset.yaml" at <eq .Values.receive.replicaCount 1>: error calling eq: incompatible types for comparison
```

Reply:
@guilhermef -> #5549